pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/leap/LeapDriver.java

```diff
@@ -135,21 +141,18 @@ protected String getAuthToken() throws Exception {
         final AuthCredentialsDto authCredentialsDto = new AuthCredentialsDto(userName, password);
         final String requestBody = mapper.writeValueAsString(authCredentialsDto);
         log.info("getAuthToken: requestBody={}", requestBody);
-        final HttpRequest request = HttpRequest.newBuilder()
-            .uri(URI.create(uri))
-            .timeout(Duration.ofMinutes(1))
-            .header("Accept", "*/*")
-            .header("Content-Type", "application/json")
-            // .POST(BodyPublishers.ofString(requestBody))
-            .build();
+        final HttpRequest request = HttpRequest.newBuilder().uri(URI.create(uri)).timeout(Duration.ofMinutes(1))
+                .header("Accept", "*/*").header("Content-Type", "application/json")
+                .POST(BodyPublishers.ofString(requestBody)).build();
         log.info("getAuthToken: request={}", request);
         final HttpResponse<String> response = client.send(request, BodyHandlers.ofString());
         log.info("getAuthToken: response={}", response);
         log.info("getAuthToken: body={}", response.body());
         if (response.statusCode() != 200) {
             throw new RuntimeException(MessageFormat.format("HTTP server returned status code {0} for request {1}",
-                response.statusCode(), request));
-        };
+                    response.statusCode(), request));
+        }
+        ;
```

Comment:
Remove `;`. It is not needed after the if block.

Reply:
Done